selenium_tests/page_objects/about_page.py

```diff
@@ -19,42 +18,89 @@ def __init__(self, driver):
     __error_message = (By.XPATH, "//div[@id='comp_7f208c379468439f140b97772ac3b622']/p[3]/*")
     __error_wrong_email = (By.XPATH, "//div[@id='comp_7f208c379468439f140b97772ac3b622']/p[2]/*")
     __error_captcha = (By.XPATH, "//div[@id='comp_7f208c379468439f140b97772ac3b622']/p[4]/*")
+    __error_captcha_single = (By.XPATH, "//div[@id='comp_7f208c379468439f140b97772ac3b622']/p[1]/*")
     __phone_number = (By.XPATH, "//div[@class='address_cont-page']/div/a")
 
-    def click_contacts_page(self):
-        self._click(self.__click_contacts_page)
-        return self
-
     def set_name(self, name):
+        """
+        Set name in field name
+        :param name: name
+        :return: self
+        """
         self._send_keys(self.__name_input, name)
         return self
 
     def set_email(self, email):
+        """
+         Set name in field email
+         :param name: email
+         :return: self
+         """
         self._send_keys(self.__email_input, email)
         return self
 
     def set_message(self, message):
+        """
+         Set message in field name
+         :param name: message
+         :return: self
+         """
         self._send_keys(self.__message_input, message)
         return self
 
     def click_submit_button(self):
+        """
+        Click on button sumbit page
+        :return: self
+        """
         self._click(self.__click_submit_button)
         return self
 
     def is_phone_number_clickable(self):
+        """
+        Wait for element would be visible
+        :return: True or False
```

Comment:
much better to use annotations!!

def is_phone_number_clickable(self) -> bool: